config/configtls/configtls.go

```diff
@@ -41,12 +40,21 @@ type TLSSetting struct {
 	// (optional)
 	CAFile string `mapstructure:"ca_file"`
 
+	// In memory PEM encoded cert. (optional)
+	CAPem []byte `mapstructure:"ca_pem"`
```

Comment:
On second thought, this should be opaque, either using [`configopaque.String`](https://pkg.go.dev/go.opentelemetry.io/collector@v0.77.0/config/configopaque#String) or adding a new alias for a byte slice